src/separations.cc

```diff
@@ -52,15 +52,34 @@ typedef std::map<std::string, Stream> StreamSet;
 
 void Separations::EnterNotify() {
   cyclus::Facility::EnterNotify();
+  std::map<int, double> efficiency_;
+
   StreamSet::iterator it;
+  std::map< int, double>::iterator it2;
+
+
   for (it = streams_.begin(); it != streams_.end(); ++it) {
     std::string name = it->first;
     Stream stream = it->second;
     double cap = stream.first;
     if (cap >= 0) {
       streambufs[name].capacity(cap);
     }
+    
+    for( it2 = stream.second.begin(); it2 != stream.second.end(); it2++  ){
+      efficiency_[it2->first] += it2->second;
+    }
+    
   }
+  
+  for( it2 = efficiency_.begin(); it2 != efficiency_.end(); it2++  ){
+    if( it2->second > 1){
+      std::stringstream ss;
+      ss << "In " << prototype() << ": Nuclei " << it2->first << " has a cumulativ seperation efficiency greater than 1 ";
+      throw cyclus::ValueError(ss.str());
```

Comment:
What if more than 1 nuclide has eff > 1?  Should we build up the stringstream with all the nuclides and then throw the exception after the loop?